src/firefly/js/ui/Stacker.jsx

```diff
@@ -6,13 +6,13 @@ import {node, shape, object, number, string, oneOf} from 'prop-types';
  * A convenience component built around Sheet and Stack to provide a <start main end> layout
  * with support for variant and color.  Use slopProps to customize
  */
-export function Stacker({variant, color, orientation='horizontal', endDecorator, startDecorator, gap=1, slotProps, children, ...props}) {
+export function Stacker({variant, color, orientation='horizontal', endDecorator, startDecorator, gap=1, slotProps, children, sx, ...props}) {
 
     const component = (variant || color) && Sheet;
     const dir = orientation === 'horizontal' ? 'row' : 'column';
     return (
         <Stack variant={variant} color={color} component={component}
-               direction={dir} spacing={gap} alignItems='center'
+               direction={dir} spacing={gap} alignItems='center' sx={sx}
```

Comment:
sx should be passed to the main Stack and not consumed by root.  To set sx in root, use slotProps.root.sx instead.

Reply:
Sorry, that was just there as an experiment to understand what was going on.